Sources/URLSessionDecodable/URLSessionDecodable.swift

```diff
@@ -23,11 +23,15 @@ extension URLSession {
         with url: URL,
         method: HTTPMethod,
         parameters: ParametersEncoding?,
+        bodyParameters: ParametersEncoding? = nil,
```

Comment:
Nit: Ideally we should keep a consistency here. Either all optional parameters with default values or none